examples/tonic-key-value-store/Cargo.toml

```diff
@@ -9,16 +9,16 @@ license = "MIT"
 [dependencies]
 bytes = "1"
 hyper = { version = "0.14.4", features = ["full"] }
-prost = "0.7"
+prost = "0.8"
 structopt = "0.3.21"
 tokio = { version = "1.2.0", features = ["full"] }
 futures = "0.3"
-tokio-stream = { version = "0.1", features = ["sync"] }
-tonic = "0.4"
+tokio-stream = { version = "0.1", features = ["sync", "net"] }
+tonic = "0.5"
```

Comment:
i think the prost and tonic updates are breaking changes, right? would have preferred to bump them in a separate PR (but it's not terribly important)

Reply:
Yeah I probably should have. CI complained about the RUSTSEC in prost so just went ahead and did it here.

Reply:
Made [a new PR](https://github.com/tower-rs/tower-http/pull/133) and reverted the update here.